src/components/Chat/BottomSheetAttachment.js

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import {View, Text, StyleSheet, TouchableNativeFeedback, ActivityIndicator} from 'react-native';
+
+import {fonts} from '../../utils/fonts';
+import {colors} from '../../utils/colors';
+import {BottomSheet} from '../BottomSheet';
+import MediasIcon from '../../assets/icons/images/medias.svg';
+import GIFIcon from '../../assets/icons/images/gif.svg';
+import FileIcon from '../../assets/icons/images/file.svg';
+import CameraIcon from '../../assets/icons/images/camera.svg';
+
+const BottomSheetAttachment = React.forwardRef((props, ref) => (
+  <BottomSheet ref={ref} closeOnPressMask={true} height={300} viewstyle={styles.container}>
+    <View style={styles.containerBottomSheet}>
+      <TouchableNativeFeedback onPress={() => props.onOpenMedia()}>
+        <View style={styles.card}>
+          <View style={styles.wrapCardImage}>
+            <MediasIcon width={20} height={20} fill={colors.black} />
+            <Text style={styles.textCard}>Send Photos & Videos</Text>
+          </View>
+          {props.isLoadingUploadMedia ? <ActivityIndicator size="small" color="#0000ff" /> : null}
+        </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `onOpenMedia` is called directly in the `onPress` prop without checking if it's defined or not. This could lead to a crash if `onOpenMedia` is undefined. The same applies for `onOpenGIF`, `onOpenCamera`, and `onOpenFile` functions on lines 24, 33, and 43 respectively. It would be safer to check if these props are actually functions before calling them.
```diff
- <TouchableNativeFeedback onPress={() => props.onOpenMedia()}>
+ <TouchableNativeFeedback onPress={() => props.onOpenMedia && props.onOpenMedia()}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
No need this because the props is required